<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExpertEmail</class>
 <widget class="QWidget" name="ExpertEmail">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>589</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ExpertEmail</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>30</y>
     <width>281</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(170, 255, 255);
font: 24pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>  审稿专家使用</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="edit_data">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>200</y>
     <width>591</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>稿件内容：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>待审稿件：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="open_td">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;
background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>打开待审稿件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="over_td">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>130</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;
background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>录用</string>
   </property>
  </widget>
  <widget class="QPushButton" name="over_td_2">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>130</y>
     <width>101</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;Arial&quot;;
background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string>不录用</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>151</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
